packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -344,28 +346,29 @@ export const distributeProportionalSharesWithLimits = (
         );
         collateralLeft = AmountMath.subtract(collateralLeft, collateralShare);
         addRemainderInCurrency(collateralShare);
-        const collateralShareRecord = { Collateral: collateralShare };
-        transfers.push([collateralSeat, seat, collateralShareRecord]);
+        transfers.push([collateralSeat, seat, { Collateral: collateralShare }]);
       }
     }
   }
 
-  // TODO(#7117) The leftovers should go to the reserve, and should be visible.
   transfers.push([currencySeat, reserveSeat, { Currency: currencyLeft }]);
 
-  // There will be multiple collaterals, so they can't all use the same keyword
-  transfers.push([
-    collateralSeat,
-    reserveSeat,
-    { Collateral: collateralLeft },
-    { [collateralKeyword]: collateralLeft },
-  ]);
+  if (!AmountMath.isEmpty(collateralLeft)) {
+    // There will be multiple collaterals, so they can't all use the same keyword
```

Comment:
nit: this comment applies to line 362

Reply:
The corresponding comment was moved to the param. I missed this one.